### ISSUE TITLE
EMI showing the wrong ingot in its unified recipes

### ISSUE THREAD

**Issue by Leclowndu93150** [AlmostReliable/almostunified#74]
### Minecraft Version

1.20.1

### Mod Version

0.9.2

### Describe the bug!

![image](https://github.com/AlmostReliable/almostunified/assets/73310949/376f257c-6a4f-4309-bb9f-c3e8c1bb9259)
![image](https://github.com/AlmostReliable/almostunified/assets/73310949/e4d70739-b851-4c68-92f9-60cbf8fc0dac)
![image](https://github.com/AlmostReliable/almostunified/assets/73310949/d6856960-a795-4536-a6dd-4ab227dba849)
ingot from Nuclearcraft
![image](https://github.com/AlmostReliable/almostunified/assets/73310949/adc83597-13cd-45a2-b792-dc5a073cca04)
ingot from enderio

### Additions and Modifications

Yes

### Additions and Modifications Description

Nomifactory CEu: Modpack 

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by rlnt** [comment]
We can't fix that because EMI has no option to hide items completely. Their in-recipe tag lookup feature and the item index are separate systems. We currently only hide unified items from the index.

Since EMI shows the first entry of a tag within a recipe that accepts it, and there is no option to specify the display stack or change the order of the entries, we can't do much here.

Before that, we used their item disabling feature which was mentioned in the initial EMI support issue submission: https://github.com/AlmostReliable/almostunified/issues/39#issuecomment-1943026014

But sadly, this ended up breaking transfer handlers for hidden items: https://github.com/AlmostReliable/almostunified/issues/67

Our interpretation of how hiding should work differs from the features EMI offers. Unlike JEI and REI, disabling the stack disabled the full functionality of EMI for the given stack altogether including transfer handlers. So we switched back to the less restrictive hiding approach (index only) in the more recent release of Almost Unified.

We talked to Emi about that and there is no easy solution to that problem without performance impact and maintenance burden on their side. I don't have a good alternative for that yet.
